core/src/com/unciv/ui/utils/UnitGroup.kt

```diff
@@ -125,7 +126,9 @@ class UnitGroup(val unit: MapUnit, val size: Float): Group() {
         }
 
         if (unit.health < 100) { // add health bar
-            addActor(ImageGetter.getHealthBar(unit.health.toFloat(), 100f, size))
+            val hp = ImageGetter.getHealthBar(unit.health.toFloat(), 100f, size*0.78f)
```

Comment:
What's the magic 0.78?

Reply:
This is similar multiplier to FlagBackground `innerMultiplier`. Only 0.1 smaller, so that HP bar width is a bit smaller than flag's inner circle